storage_controller/src/timeline_import.rs

```diff
@@ -28,7 +28,7 @@ impl ShardImportStatuses {
         ShardImportStatuses(
             shards
                 .into_iter()
-                .map(|ts_id| (ts_id, ShardImportStatus::InProgress))
+                .map(|ts_id| (ts_id, ShardImportStatus::InProgress(None)))
```

Comment:
https://neondb.slack.com/archives/C0277TKAJCA/p1743615626575609?thread_ts=1743525492.648389&cid=C0277TKAJCA

```suggestion
                .map(|ts_id| (ts_id, ShardImportStatus::InProgress(None::<ShardImportProgress>)))
```

Reply:
This is one of the places where you can actually tell what's going on fairly easily IMO, but let's get in the habit of being explicit.

Reply:
https://github.com/neondatabase/neon/pull/11862/commits/3ceb7c2d3610ac732df333c7fa09ea901e4a5fb6